### PR TITLE
Release script

### PR DESCRIPTION
**Pull request by timowest**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+git checkout master
+git pull
+mvn versions:set -DgenerateBackupPoms=false
+mvn clean deploy -Dtest=X -Dgpg.skip=false
```
Comment (johnktims) [inline release.sh]: What does `-Dtest=X` do?

Comment (timowest) [inline release.sh]: It skips the tests, but I'll change it to `-DskipTests`.

**Comment by johnktims** [comment]
Looks fine to me.

**Comment by Shredder121** [comment]
Oops, looks fine to me.

**Pull request merged**